lib/rules/declaration-property-value-no-unknown/README.md

```diff
@@ -24,6 +24,8 @@ It sometimes overlaps with:
 
 If duplicate problems are flagged, you can turn off the corresponding rule.
 
+For more details on CSS value definition syntax, refer to the [CSSTree Syntax Reference](https://csstree.github.io/docs/syntax/).
```

Comment:
We tend to favour the active voice in our docs. How about we put the following after the 1st line and before the "This rule is only appropriate for CSS..." one:

> You can filter the [CSSTree Syntax Reference](https://csstree.github.io/docs/syntax/) to find out what value syntax is known for a property.

That'll keep the related bits of the extended description together.

Reply:
Good suggestion! I've applied it.